resources/js/components/ui/calendar/ResourcesSelector.tsx

```diff
@@ -0,0 +1,128 @@
+import {cx, htmlAttributes} from "components/utils";
+import {For, JSX, Show, VoidComponent, createMemo, createRenderEffect, createSignal, splitProps} from "solid-js";
+
+export type ResourcesSelectionMode = "radio" | "checkbox";
```

Comment:
Będzie używany w obu trybach? Jeśli nie no to wiem że różnica w podejściu, no ale bez sensu nadbudowywać żeby wspierać oba.

Wydaje się że raczej powinny być checkboxy do kalendarza.

Reply:
Już nawet jest użyte w obu trybach w tym PR. Checkbox jest dla widoku dnia, dla pozostałych jest radio. To jest jak w versum.